.github/workflows/KubernetesExternalClients.yaml

```diff
@@ -9,6 +9,10 @@ on:
         description: Default is hazelcast, but if you would like to run the workflow with your forked repo, set your github username
         required: true
         default: hazelcast
+      java_client_run:
+        description: Java Client (Standard), (old name is Java thin client) = Set the branch name of client to run, otherwise set it as 'no' in order to skip running this client
```

Comment:
the name will probably change, no standard. The platform one will be called `Platform Java Client` ([comment](https://hazelcast.atlassian.net/wiki/spaces/EN/pages/5213650990/ADR-00027-+Type+2+-Java+Client+Naming+and+Licensing#Considered-Options))

Reply:
yes, let make it Java Client and let me remove (old name is Java thin client) as well

Reply:
https://github.com/hazelcast/client-compatibility-suites/pull/128/commits/b35e95514d0a344228bb2b41c0dbe4aaa6a4ebc0